src/rust-2018/module-system/path-clarity.md

```diff
@@ -268,21 +203,21 @@ see their names, instead of having a bunch of tabs named `mod.rs`.
 
 # Uniform paths
 
-> Uniform paths are a nightly-only feature.
+![Minimum Rust version: 1.32](https://img.shields.io/badge/Minimum%20Rust%20Version-1.32-brightgreen.svg)
 
 The uniform paths variant of Rust 2018 simplifies and unifies path handling
```

Comment:
Not part of your diffs, but we should probably not talk about a "variant" anymore since the anchored paths proposal was rejected.